model/src/main/java/org/mskcc/oncotree/model/Version.java

```diff
@@ -50,6 +51,9 @@ public class Version implements Serializable {
     @JsonProperty("visible")
     private boolean visible;
 
+    @JsonProperty("release_date")
+    private String releaseDate;
```

Comment:
@sheridancbio This is because it was getting through in additional properties before and the way it works now it is explicitly set.  I needed a setter and a variable etc.  I did add it to the JsonPropertyOrder as you suggested.

Reply:
👍